client/js/components/AutheticationLoadingScreen/AuthenticationLoadingScreen.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
```

Comment:
your file name is AuthenticationLoadingScreen. Please put it in the screens dir. 
Also, use your index.js file for this screen.